checks/zk_quorum.go

```diff
@@ -0,0 +1,129 @@
+package checks
+
+import (
+    "bufio"
+    "fmt"
+	"time"
```

Comment:
Agreed. Will format this concrete line by hands, then run `go fmt` after merge this request (we have a lot of unformatted code, and it just smash PR with unrelated changes)